test/mocha.opts

```diff
@@ -0,0 +1 @@
+--timeout 120000
```

Comment:
I think we can pass this timeout in package.json on this [line](https://github.com/strongloop/loopback-connector-oracle/pull/128/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R17). And in that case, no need for this file anymore.